### PR TITLE
Configurable stiffnesses in MouseDrag

### PR DESCRIPTION
**Pull request by Henrique-BO**
# 🎉 New feature

Requires #2038 

## Summary

Makes the rotation and translation stiffnesses in the MouseDrag plugin configurable.

This can be done either through the `<rotation_stiffness>` and `<position_stiffness>` elements on the SDF file or through fields on the interface. 

image

## Test it

Open any test world.

Load the Mouse Drag plugin.

Modify the stiffness values and observe the effects when dragging links.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [x] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Review by ahcorde — DISMISSED** [review]

File: src/gui/plugins/mouse_drag/MouseDrag.cc [hunk 1]
```diff
@@ -0,0 +1,754 @@
...
+      req.mutable_entity()->set_id(worldEntity);
+      auto plugin = req.add_plugins();
+      plugin->set_name(name);
+      plugin->set_filename(filename);
+      plugin->set_innerxml(innerxml);
+
+      msgs::Boolean res;
+      bool result;
+      unsigned int timeout = 5000;
+      std::string service{"/world/" + this->dataPtr->worldName +
```
[the remark this answers is not in the document]
Comment (Henrique-BO) [reply src/gui/plugins/mouse_drag/MouseDrag.cc]: Done in 60f788f

File: src/gui/plugins/mouse_drag/MouseDrag.qml [hunk 2]
```diff
@@ -23,23 +23,72 @@ GridLayout {
...
     color: "dimgrey"
-    text: qsTr("Ctrl+Left-Click: rotation")
+    text: qsTr("Rotation (Ctrl+Left-Click)")
+  }
+
+  Label {
+    Layout.columnSpan: 2
+    horizontalAlignment: Text.AlignRight
+    id: rotStiffText
+    color: "dimgrey"
```
Comment (azeey) [inline src/gui/plugins/mouse_drag/MouseDrag.qml:51]: nit: The `dimgrey` makes it look like these options are disabled. Can we just use the same color as used for " Apply force to center of mass"?

**Review by azeey — APPROVED** [review]
LGTM! Just one nit.

File: src/gui/plugins/mouse_drag/MouseDrag.qml (hunk 2 above)
Comment (Henrique-BO) [reply src/gui/plugins/mouse_drag/MouseDrag.qml]: Done in 3a6c617

**Pull request merged**